ui/src/index.ts

```diff
@@ -131,7 +131,7 @@ window.addEventListener("load", async function () {
     const stats = await getDocs(
       query(collection(firestore, collectionName), orderBy("campaign", "desc"))
     );
-    return stats.docs.map(snapshot => ({
+    return stats.docs.map((snapshot: any) => ({
```

Comment:
What makes this needed now but not before? Can't TypeScript infer the type of snapshot from the type of stats?

Reply:
I had warning about it to write explicitly "any"